buttons.h

```diff
@@ -62,11 +73,11 @@ class Button{
             // button color
             this->_color = btn_color(c);
             // load button color image
-            this->_pic = load_bitmap(to_string(c), this->_color);
+            //this->_pic = load_bitmap(to_string(c), this->_color);
             // create sprite from image
-            this->_btn = create_sprite(this->_pic);
+            this->_btn = create_sprite(this->_color);
             // add hightlight layer to sprite
-            sprite_add_layer(this->_btn, this->_hghlt, this->_hightlight);
+            sprite_add_layer(this->_btn, this->_hghlt, this->_highlight);
```

Comment:
I think a lot of these changes are because your branch status is quite a few iterations behind current develop. Try getting your branch up-to-date with main and then re-attempting this pull request. Thanks Huy!